src/routes/committee/suggestions/+page.svelte

```diff
@@ -137,4 +176,68 @@
 	.titleText {
 		font-size: x-large;
 	}
+
+	:global(.addButton) {
+		border-color: rgba(41, 252, 101, 255) !important;
+		background-color: rgba(0, 0, 0, 0) !important;
+		color: rgba(41, 252, 101, 255) !important;
+	}
+
+	:global(.ilanButton) {
+		animation: rainbow 2.5s linear !important;
+		animation-iteration-count: infinite !important;
+		background-color: rgba(0, 0, 0, 0) !important;
+	}
+
+	@keyframes rainbow {
```

Comment:
Is het mogelijk deze te importen oid? Dit clogged een beetje de file

Reply:
Ik wilde bijna vragen kan dat met css... ood moment